OpenEdXMobile/src/main/java/org/edx/mobile/base/FindCoursesBaseActivity.java

```diff
@@ -110,6 +110,7 @@ private void setupWebView() {
 
         client.setActionListener(this);
         client.setPageStatusListener(this);
+        webview.getSettings().setDomStorageEnabled(true);
```

Comment:
The change on the ECOM side was only applied to the marketing site. 

Additionally, the Find Courses page does not use EdxWebView. For the hotfix, I'm going to merge this but I will open up a follow up PR that we can spend more time on.